### ISSUE TITLE
🐛 [Bug report]: Unnecessary Multiple Re-rendering making multiple graphql call

### ISSUE THREAD

**Issue by Angryman18** [reactplay/react-play#347]
**Describe the bug**

**Expected behavior**
multiple unnecessary re-rendering and graphql calls are observed.

**Screenshots**
![Screenshot from 2022-06-26 10-36-11](https://user-images.githubusercontent.com/63530626/175800228-c3d7b4e1-073b-4fb1-b714-676c7455fdd4.png)

**Comment by 6km** [comment]
do these requests have the same query payload?

**Comment by priyankarpal** [comment]
yeah! noticed that problem. any updates @Angryman18 for this issue?

**Comment by Angryman18** [comment]
Im kinda out of context for now about what is the issue @priyankarpal

**Comment by priyankarpal** [comment]
:)
